build/en/docs/index.njk

```diff
@@ -212,10 +273,19 @@ rootClass: docs
       To optimize videos:
     </p>
     <ul class="real-ul">
-      <li>Trim to the exact length you need
+      <li>Trim to the exact length you need</li>
       <li>Crop to only the visible portion of our video (usually vertical, 9:16)
-      <li>Encode with the highest compression yielding acceptable results. For H264, that’s often below a CRF of 23 or a bit rate below 1500kbps.</li>
+      <li>Encode with the highest compression yielding acceptable results. For H264, that’s often below a CRF of 23 or a bit rate below 1500kbps</li>
     </ul>
+    <h2 id="Monetize-your-stories">Monetize your stories</h2>
+    <p>
+    Make money off your Web Stories by programmatically inserting ads. You can enable the following supported integrations in your plugin's settings:
```

Comment:
I would maybe choose a different word than `programmatically` here, as that is a certain term for a type of ad. Maybe something like "...by automatically inserting ads between pages."